jans-cedarling/cedarling/src/init/policy.rs

```diff
@@ -0,0 +1,139 @@
+/*
+ * This software is available under the Apache-2.0 license.
+ * See https://www.apache.org/licenses/LICENSE-2.0.txt for full text.
+ *
+ * Copyright (c) 2024, Gluu, Inc.
+ */
+
+use std::collections::HashMap;
+
+use base64::prelude::*;
+use cedar_policy::PolicyId;
+
+// we use camel case to show that it is like a constant
+#[allow(non_camel_case_types)]
+#[derive(Debug, thiserror::Error)]
+enum ParsePolicySetErrMsg {
+    #[error("unable to decode base64")]
+    BASE64,
+    #[error("unable to decode to utf8 string")]
+    STRING,
+    #[error("unable to decode from human redable format")]
+    HUMAN_REDABLE,
+    #[error("could not collect policy store's to policy set")]
+    CREATE_POLICY_SET,
+}
+
+/// Represents a raw data of the `Policy` in the `PolicyStore`
+/// is private and used only in the [`parse_policy_set`] function
+#[derive(Debug, serde::Deserialize)]
+struct PolicyRaw {
```

Comment:
why not `RawPolicy`? I'm fine with either, but I think `RawPolicy` will be more descriptive?

Reply:
done